bitcode/devicelib.cl

```diff
@@ -847,14 +847,12 @@ EXPORT uint __chip_float_as_uint(float x) { return as_uint(x); }
 EXPORT long __chip_double_as_longlong(double x) { return as_long(x); }
 EXPORT double __chip_longlong_as_double(long int x) { return as_double(x); }
 
-#ifdef CHIP_ENABLE_NON_COMPLIANT_DEVICELIB_CODE
-
 // See c_to_opencl.def for details.
-#define DEF_UNARY_FN_MAP(NAME_, TYPE_)                                         \
-  TYPE_ MAP_PREFIX##NAME_(TYPE_ x) { return NAME_(x); }
-#define DEF_BINARY_FN_MAP(NAME_, TYPE_)                                        \
-  TYPE_ MAP_PREFIX##NAME_(TYPE_ x, TYPE_ y) { return NAME_(x, y); }
+#define MAP_PREFIX __chip_c2ocl_
+#define DEF_UNARY_FN_MAP(FROM_FN_, TO_FN_, TYPE_)                              \
+  TYPE_ __chip_c2ocl_##FROM_FN_(TYPE_ x) { return TO_FN_(x); }
+#define DEF_BINARY_FN_MAP(FROM_FN_, TO_FN_, TYPE_)                             \
+  TYPE_ __chip_c2ocl_##FROM_FN_(TYPE_ x, TYPE_ y) { return TO_FN_(x, y); }
 #include "c_to_opencl.def"
 #undef UNARY_FN
 #undef BINARY_FN
```

Comment:
Can you remind me why we have this here? If we already have this in c_to_opencl.c which we compile and then link into the final .bc?

Reply:
The reason is explained in the [c_to_opencl.def](https://github.com/CHIP-SPV/chipStar/blob/0dba21d5c36121b9f0323acb2319e2bbcb00fc83/bitcode/c_to_opencl.def#L23-L39).